docs/management_api.yml

```diff
@@ -559,6 +559,11 @@ definitions:
   PreAuthSet:
     type: object
     properties:
+      force:
+        type: boolean
+        description: >
+          Indicates that despite the device having accepted authsets the preauthorization
```

Comment:
Without this flag, device with any authset (accepted, pending, rejected) will prevent from preauthorization.

Reply:
fixed.